src/pages/VenuePage/VenuePage.tsx

```diff
@@ -230,7 +230,7 @@ const VenuePage = () => {
       <WithNavigationBar>
         <AuthenticationModal
           show={true}
-          onHide={() => {}}
+          onHide={() => { }}
```

Comment:
Can we remove this/make it optional rather than a noop func? In the meantime, could extract as a `const noopHandler = () => {}` outside of the component